docs/README.md

```diff
@@ -48,14 +48,13 @@ for example, if you had the follwing Secret defined in Kubernetes:
 {% include_relative examples/username-pass.yaml %}
 {% endhighlight %}
 
-you could use it via the [credentials binding plugin](https://plugins.jenkins.io/credentials-binding) 
+you could use it via the [Credentials Binding plugin](https://plugins.jenkins.io/credentials-binding) 
 
-```
+```groovy
```

Comment:
this caused page build issues... it appears as though GH pages does not like this. fixed in be22e0f8aee86a24e832a86fe78564c82c402788